episodes/chapter5.md

```diff
@@ -0,0 +1,302 @@
+
+[
+    {'address1': '', 'studies': '/api/study?organization_id=3', 'users': '/api/user?organization_id=3', 'zipcode': '', 'domain': '', 'tasks': '/api/task?init_org_id=3', 'public_key': '', 'name': 'CANTABRIA_organization', 'address2': '', 'nodes': '/api/node?organization_id=3', 'country': 'Spain', 'collaborations': '/api/collaboration?organization_id=3', 'runs': '/api/run?organization_id=3', 'id': 3}, 
+    {'address1': '', 'studies': '/api/study?organization_id=4', 'users': '/api/user?organization_id=4', 'zipcode': '', 'domain': '', 'tasks': '/api/task?init_org_id=4', 'public_key': '', 'name': 'LIFELINES_organization', 'address2': '', 'nodes': '/api/node?organization_id=4', 'country': 'The Netherlands', 'collaborations': '/api/collaboration?organization_id=4', 'runs': '/api/run?organization_id=4', 'id': 4}, 
+    {'address1': '', 'studies': '/api/study?organization_id=5', 'users': '/api/user?organization_id=5', 'zipcode': '', 'domain': '', 'tasks': '/api/task?init_org_id=5', 'public_key': '', 'name': 'GAZEL_organization', 'address2': '', 'nodes': '/api/node?organization_id=5', 'country': 'France', 'collaborations': '/api/collaboration?organization_id=5', 'runs': '/api/run?organization_id=5', 'id': 5}, 
+    {'address1': '', 'studies': '/api/study?organization_id=7', 'users': '/api/user?organization_id=7', 'zipcode': '', 'domain': '', 'tasks': '/api/task?init_org_id=7', 'public_key': '', 'name': 'PhY24-consortium', 'address2': '', 'nodes': '/api/node?organization_id=7', 'country': '', 'collaborations': '/api/collaboration?organization_id=7', 'runs': '/api/run?organization_id=7', 'id': 7}
+]
+```
+
+Getting the status of an organization node for a given collaboration: The following returns the nodes (there is a node for each collaboration the organization contributes to).
+
+``` python
+>>> client.node.list(organization=4)
+
+[
+    {'config': [], 'last_seen': '2024-06-18T06:57:48.114072', 
+    'organization': {'id': 4, 'link': '/api/organization/4', 
+    'methods': ['PATCH', 'GET']}, 
+    'collaboration': {'id': 4, 'link': '/api/collaboration/4', 
+    'methods': ['GET', 'PATCH', 'DELETE']}, 
+    'name': 'PhY-research - LIFELINES_organization', 'status': 'offline', 
+    'type': 'node', 'ip': None, 'id': 13}, 
+    {'config': [], 'last_seen': None, 
+    'organization': {'id': 4, 'link': '/api/organization/4', 
+    'methods': ['PATCH', 'GET']}, 
+    'collaboration': {'id': 5, 'link': '/api/collaboration/5', 
+    'methods': ['GET', 'PATCH', 'DELETE']}, 
+    'name': 'GHT_consortium - LIFELINES_organization', 'status': None, 
+    'type': 'node', 'ip': None, 'id': 18}
+]
+```
+
+:::::::::::::::::::::::::::::::::
+
+::::::::::::::::::::::::::::::::::::: challenge
+## Challenge 3: Creating a task that runs a partial algorithm**
+
+The following is the information of a new algorithm **(TODO a new algorithm should be created for this)**, that you need to use on the collaboration whose 100% of its nodes are active:
```

Comment:
Could you add this to the issue tracker?

Reply:
Issue #36 created.